MONEY/NetworkSpentSupplyContactService.md

```diff
@@ -0,0 +1,20 @@
+---
+ns: MONEY 
+---
+
+## _NETWORK_SPENT_SUPPLY_CONTACT_SERVICE
+
+```c
+// 0xEBD482B82ACB8BAD 
+void _NETWORK_SPENT_SUPPLY_CONTACT_SERVICE(Any p0, Any p1, Any p2, Any p3);
+```
+
+```
+NativeDB Introduced: v2545
+```
+
+## Parameters
+* **p0**:
+* **p1**:
+* **p2**:
+* **p3**:
```

Comment:
Param's seems to be BOOL `amount`, BOOL `if from bank`, BOOL `if from bank AND wallet` and `npcprovider`.